src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -760,20 +760,23 @@ public DatasetSummaryModel updateTags(UUID id, TagUpdateRequestModel tagUpdateRe
     return datasetDao.retrieveSummaryById(id).toModel();
   }
 
-  public String enableInheritSteward(UUID datasetId, AuthenticatedUserRequest userReq) {
-    String description = "Enable InheritSteward for dataset " + datasetId;
+  public String setInheritSteward(
+      UUID datasetId, boolean inheritSteward, AuthenticatedUserRequest userReq) {
+    String description =
+        String.format("Set InheritSteward for Dataset, %s, to %s", datasetId, inheritSteward);
```

Comment:
It's minor but I would avoid an appositive here. Also other messages only capitalize the first word.
```suggestion
        String.format("Set inherit steward to %s for dataset %s", inheritSteward, datasetId);
```